.github/actions/build/action.yaml

```diff
@@ -26,14 +27,18 @@ runs:
         else
           BUILD_PLATFORM=${COBALT_EVERGREEN_LOADER}
           BUILD_TARGET='loader_app_install elf_loader_sandbox_install native_target/crashpad_handler'
```

Comment:
Can you inline `${{ matrix.evergreen_loader_extra_build_targets }}` in this line instead?

Reply:
Yes, I could, I was just trying to follow the same pattern used for `extra_gn_arguments` in `.github/actions/build/action.yaml`. That said, would you still prefer inline? Please let me know and I will update accordingly.